### PR TITLE
net-libs/nodejs: keyword 16.14.1 for ~riscv

### PR DESCRIPTION
**Pull request by ArchFeh**
> test/parallel/test-child-process-spawn-argv0.js
test/parallel/test-dgram-multicast-set-interface.js
test/parallel/test-memory-usage.js
test/parallel/test-memory-usage-emfile.js
test/parallel/test-setproctitle.js
test/parallel/test-worker-memory.js

above 6 tests failed in QEMU environment, but can pass on SiFive Unmatched boards.
In Unmatched boards, with FEATURES=network-sandbox will cause below one test failure

> test-dns-setserver-when-querying

Closes: https://bugs.gentoo.org/782913
Signed-off-by: Yu Gu <guyu2876@gmail.com>

### CODE REVIEW

File: net-libs/nodejs/nodejs-16.13.1.ebuild [hunk 1]
```diff
@@ -49,6 +49,7 @@ DEPEND="${RDEPEND}"
 PATCHES=(
 	"${FILESDIR}"/${PN}-12.22.5-shared_c-ares_nameser_h.patch
 	"${FILESDIR}"/${PN}-15.2.0-global-npm-config.patch
+	"${FILESDIR}"/${PN}-16.13.1-riscv-arch.patch
```
Comment (jsmolic) [inline net-libs/nodejs/nodejs-16.13.1.ebuild:52]: The last commit is adding back this patch which was dropped 2 commits before that, can you squash this change please (i.e. don't drop the patch at all)

Comment (guyu2876) [reply net-libs/nodejs/nodejs-16.13.1.ebuild:52]: Thx for reminding. Done.

**Comment by williamh** [comment]
@gentoo/risc-v

**Comment by ajakk** [comment]
cc @gentoo/risc-v

**Comment by dlan17** [comment]
how did you test? I've got a test failure, with command: USE="test" FEATURES="test -buildpkg" ALLOW_TEST="network" emerge nodejs

[----------] Global test environment tear-down
[==========] 127 tests from 21 test suites ran. (14459 ms total)
[  PASSED  ] 126 tests.
[  FAILED  ] 1 test, listed below:
[  FAILED  ] DebugSymbolsTest.ReqWrapList

 1 FAILED TEST
 * ERROR: net-libs/nodejs-16.14.1::gentoo failed (test phase):

**Comment by guyu2876** [comment]
> 

Yes it's a known bug with >gcc 10. https://github.com/nodejs/node/issues/33899
I added 
> CFLAGS="-fno-strict-aliasing $CFLAGS" 
CXXFLAGS="-fno-strict-aliasing $CXXFLAGS"
to my test.conf because I don't know should we add it in the ebuild to avoid this test failure.

**Pull request closed without merging**